torch/csrc/cuda/Event.h

```diff
@@ -2,9 +2,10 @@
 #define THCP_EVENT_INC
 
 #include <ATen/cuda/CUDAEvent.h>
+#include <torch/csrc/Event.h>
 #include <torch/csrc/python_headers.h>
 
-struct THCPEvent {
+struct THCPEvent : THPEvent {
   PyObject_HEAD at::cuda::CUDAEvent cuda_event;
```

Comment:
Do you actually need the PyObject_HEAD after subclassing here?

Reply:
It is not necessary. Removed.